.nvmrc

```diff
@@ -0,0 +1 @@
+v18.7.0
```

Comment:
FYI: this is for automatic version switching when using [nvm](https://github.com/nvm-sh/nvm) or [fvm](https://github.com/Schniz/fnm#--fast-node-manager-fnm----)